UIKitDZ/UIKitDZ/ViewControllers/SecondViewController.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  SecondViewController.swift
+//  UIKitDZ
+//
+//  Created by coder on 24.09.2022.
+//
+
+import Foundation
+import UIKit
+/// SecondViewController - второй экран с информацией
+class SecondViewController: UIViewController {
+    
+    let firstImage: UIImageView = {
+        let imageFirst = UIImageView(frame: CGRect(
+            x: 20,
+            y: 120,
+            width: 70,
+            height: 70))
+        imageFirst.image = UIImage(named: "default")
+        imageFirst.layer.cornerRadius = 30
+        imageFirst.clipsToBounds = true
+        return imageFirst
+    }()
+    let secondImage: UIImageView = {
+        let imageSecond = UIImageView(frame: CGRect(
+            x: 20,
+            y: 220,
+            width: 70,
+            height: 70))
+        imageSecond.image = UIImage(named: "default")
+        imageSecond.layer.cornerRadius = 30
+        imageSecond.clipsToBounds = true
+        return imageSecond
+    }()
+    let threedImage: UIImageView = {
+        let imageThreed = UIImageView(frame: CGRect(
+            x: 20,
+            y: 320,
+            width: 70,
+            height: 70))
+        imageThreed.image = UIImage(named: "default")
+        imageThreed.layer.cornerRadius = 30
+        imageThreed.clipsToBounds = true
+        return imageThreed
+    }()
+    let labelJack = UILabel()
+    let labelJackInfo = UILabel()
+    let labelJackDays = UILabel()
+    
+    let labelJeremy = UILabel()
+    let labelJeremyInfo = UILabel()
+    let labelJeremyDays = UILabel()
+    
+    let labelGarfild = UILabel()
+    let labelGarfildInfo = UILabel()
+    let labelGarfildDays = UILabel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        buttonBack()
```

Comment:
```suggestion
        configureViews()
```
складываем сюда методы
   buttonBack()
        settingsView()
        addImageOnView()
        settingsLabelsForSecondController()
        buttonAdd()